packages/db/src/scripts/import-bible.ts

```diff
@@ -55,12 +55,9 @@ async function run() {
           },
         });
 
-        // Some entries in the words list are text in between words, so we ignore them.
-        const filteredWords = words.filter((word) => word.length === 6);
-
-        for (let wordIndex = 0; wordIndex < filteredWords.length; wordIndex++) {
-          const [text, , english, grammar, rawStrongs] =
-            filteredWords[wordIndex];
+        for (let wordIndex = 0; wordIndex < words.length; wordIndex++) {
+          const [text, , english, grammar = '', rawStrongs = '????'] =
```

Comment:
For now, words without a strongs number are linked to a single lemma, but in #248, I'm working on getting access to a datasource that can distinguish these words and their grammatical forms since Sheldon's data didn't have this.